src/__tests__/wait-for-element-to-be-removed.js

```diff
@@ -0,0 +1,334 @@
+import {waitForElementToBeRemoved} from '../'
+// adds special assertions like toBeTruthy
```

Comment:
toBeTruthy is built-into jest. This adds things like toBeInTheDocument. But I don't think we need this comment anyway. Please remove it.